core/includes/install.core.inc

```diff
@@ -1409,28 +1383,51 @@ function install_select_language(&$install_state) {
  *
  * @ingroup forms
  */
-function install_select_language_form($form, &$form_state, $files) {
+function install_select_language_form($form, &$form_state, $files, $install_state) {
   include_once BACKDROP_ROOT . '/core/includes/standard.inc';
   include_once BACKDROP_ROOT . '/core/includes/locale.inc';
 
-  $standard_languages = standard_language_list();
+  $standard_languages = install_available_translations();
   $select_options = array();
   $languages = array();
 
-  foreach ($files as $file) {
-    if (isset($standard_languages[$file->langcode])) {
-      // Build a list of select list options based on files we found.
-      $select_options[$file->langcode] = $standard_languages[$file->langcode][1];
-    }
-    else {
-      // If the language was not found in standard.inc, display its langcode.
-      $select_options[$file->langcode] = $file->langcode;
+  if (count($install_state['translations']) > 1) {
+    $files = $install_state['translations'];
+  }
+  else {
+    $files = [];
+  }
```

Comment:
This doesn't feel right to me 🤔 ...we are:
- checking for more than 1 translation, and if that's the case, then we are setting `$files` to the array with the translations 👍🏼 
- if not more than 1 translation files (which effectively means either 1 or 0), then setting `$files` to an empty array 🤔 🤔 🤔 ...hmm 🤔 ...what am I missing here?

Reply:
Removed